tests/Api/SmsesTest.php

```diff
@@ -51,4 +62,50 @@ public function testBatchEndpoints()
     {
         $this->standardTestBatchEndpoints();
     }
+
+    private function createEntity($type, $entity) {
+        $context = $this->getContext($type);
+        $response = $context->create($entity);
+        $itemName = $context->getItemName();
+
+        $this->assertTrue(isset($response[$itemName]), isset($response['error']['message']) ? $response['error']['message'] : '');
+
+        return isset($response[$itemName]) ? $response[$itemName] : false;
+    }
+
+    private function removeEntity($type, $entity) {
+        $context = $this->getContext($type);
+        $response = $context->delete($entity['id']);
+        $itemName = $context->getItemName();
+
+        $this->assertTrue(isset($response[$itemName]), isset($response['error']['message']) ? $response['error']['message'] : '');
+
+        return isset($response[$itemName]) ? true : false;
+    }
+
+    public function testSendSMS() {
+        $contact = $this->createEntity('contacts', $this->contactPayload);
+        $message = $this->createEntity('smses', [
+            'created_by'  => $contact['id'],
+            'create_by_user' => "API Test",
+            'name'      => 'API Test Message',
+            'message'   => 'You should not get this',
+        ]);
+
+        if ($contact && $message) {
+            $requestResponse =  $this->api->makeRequest($this->endpoint.'/' . $message['id'] . '/contact/' . $contact['id'] . '/send');
+            $this->assertCount(1, $requestResponse['errors']);
+            $response = $this->api->sendSMS($message['id'], $contact['id']);
+
+            $this->assertTrue(!$response['result']['sent']);
```

Comment:
Is this test expecting that the sent will fail?

Reply:
yes, it is. actually it fails for different reason than I intended but I am just to test that Twillio transport will be called. There are other tests to test the transportChain and transport calls itself.